app/addons/documents/index-editor/stores.js

```diff
@@ -186,7 +186,7 @@ Stores.IndexEditorStore = FauxtonAPI.Store.extend({
   getSaveDesignDoc: function () {
     if (this._designDocId === 'new-doc') {
       var doc = {
-        _id: '_design/' + this._newDesignDocName,
+        _id: '_design/' + encodeURIComponent(this._newDesignDocName),
```

Comment:
Here is encoded DesignDoc name in POST request to CouchDB
Where is may be encoded viewname in  POST request to CouchDB?